oam/install_scripts/mcs-writeengineserver.service.in

```diff
@@ -7,7 +7,4 @@ After=mcs-exemgr.service
 Type=simple
 ExecStart=/usr/bin/env bash -c "LD_PRELOAD=$(ldconfig -p | grep -m1 libjemalloc | awk '{print $1}') exec @ENGINE_BINDIR@/WriteEngineServer"
 Restart=on-failure
-TimeoutStopSec=2
-
-[Install]
-WantedBy=mariadb-columnstore.service
+TimeoutStopSec=20
```

Comment:
Questionable thing. We will come up with a dynamic solution.

Reply:
I suspect it's the same as worker/controller problem.  It is blocking on a read() for longer than we want, so not checking its status vars often enough.  If so, it's easy to fix that, but we don't have to worry about that in this PR.

Reply:
I suppose this comment goes to a higher snippet. And the answer is again no. The only thread that left hangs in ThreadPool::~ waiting for a non-exisiting thread to join.

Reply:
I just put TimeoutStopSec to send termkill here just in case.

Reply:
Note, by default term is called, and then once TimoutStopSec is reached kill is called so all this does is increase time for process to terminate cleanly